region/wreckedship/main.json

```diff
@@ -1986,7 +1986,12 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": [ "Gravity" ]
+                  "requires": [
+                    {"or": [
+                        "Gravity",
+                        {"not": [ "f_DefeatedPhantoon" ]}
```

Comment:
Would you mind using a separate "Unpowered" strat? The strat names are included in the Map Rando spoiler log and can be useful to clarify what's happening in case we put these unpowered strats in logic at some point.